### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by shammamah**
Add template for bug reports.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (mkcor) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
about: Report a bug to help us improve Dash Bio
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```
Comment (mkcor) [inline .github/ISSUE_TEMPLATE/bug_report.md]: How about 'starting' a list with `-` (or `*`) on the next line, like you did line 25?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (mkcor) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
If applicable, add screenshots to help explain the issue.
```

**Comment by mkcor** [comment]
Are we planning on having [multiple issue templates](https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository)?

**Comment by shammamah** [comment]
@mkcor Can you think of a use case for a different kind of issue template? Maybe we could add one for feature requests, but we should have at least this one in place to begin with.

**Comment by mkcor** [comment]
Well, no (apart from what I said privately), that's why I was wondering why you went for the multiple issue template structure, instead of creating a single `.github/issue_template.md` file.

**Comment by shammamah** [comment]
@mkcor I see what you mean. I went through this process: https://help.github.com/en/articles/creating-issue-templates-for-your-repository

**Comment by shammamah** [comment]
This also establishes a structure if we ever want to split off into different sub-types of issues in the future, based on the issues that get opened in the future (e.g., app vs. component issues).

**Comment by mkcor** [comment]
@shammamah fair enough!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (shammamah) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good catch! Fixed in ff7c72f.

**Pull request merged**